src/storage/src/store.rs

```diff
@@ -31,7 +31,7 @@ pub trait IterFutureTrait<'a, I: StateStoreIter<Item = (Bytes, Bytes)>, R, B> =
     Future<Output = StorageResult<I>> + Send;
 pub trait EmptyFutureTrait<'a> = Future<Output = StorageResult<()>> + Send;
 pub trait SyncFutureTrait<'a> =
-    Future<Output = StorageResult<(usize, Vec<LocalSstableInfo>)>> + Send;
+    Future<Output = StorageResult<(usize, Vec<LocalSstableInfo>, bool)>> + Send;
```

Comment:
Need more documentations on the meaning of each elements in the return type. I think we can have a `SyncResult` struct for the return value to make the code more readible.